src/Symfony/Component/Console/Formatter/OutputFormatterStyle.php

```diff
@@ -199,7 +199,7 @@ public function apply($text)
             $unsetCodes[] = $option['unset'];
         }
 
-        if (null !== $this->href) {
+        if (null !== $this->href && 'JetBrains-JediTerm' !== getenv('TERMINAL_EMULATOR')) {
```

Comment:
the call should be cached otherwise the impact is going to be huge, especially on vardumper where the style method is called quite often.

Reply:
Indeed. Fixed